src/ansys/dpf/composites/layup_info/material_operators.py

```diff
@@ -23,17 +23,16 @@ class MaterialOperators:
         evaluate material properties. This container cannot be
         queried in Python, but it can be passed to other DPF operators
         that evaluate the properties.
-    result_info_provider:
-        Provides the ``ResultInfo`` object.
     """
 
     material_provider: Operator
     material_support_provider: Operator
```

Comment:
Moved unit system related logic out of MaterialOperators. Now it is possible to specify the unit system independently, so it is now a separate entity that has to be passed to functions.